src/test/java/nextstep/subway/lineStation/LineStationAcceptanceTest.java

```diff
@@ -170,16 +165,100 @@ public void setUp() {
         // when
         JsonPath 신도림역 = 지하철_역_생성("신도림역");
         JsonPath 대림역 = 지하철_역_생성("대림역");
+        Map<String, Object> paramMap = 지하철_구간_추가_파라미터_생성(getId(신도림역), getId(대림역), 5);
+        ExtractableResponse<Response> postResponse = 지하철_구간_추가(getId(이호선), paramMap);
+
+        // then
+        ExtractableResponse<Response> getResponse = 노선_아이디로_지하철역_조회(getId(이호선));
+        assertThat(postResponse.statusCode()).isEqualTo(예외_발생());
+        assertThat(getResponse.jsonPath().getList("stations")).hasSize(2);
+        assertThat(getResponse.jsonPath().getInt("distance")).isEqualTo(DISTANCE);
+    }
+
+    /**
+     * Given 역 3개와 노선을 생성하고
+     * When 종점을 제거하면
+     * Then 노선에 역 2개가 남는다.
+     */
+    @DisplayName("노선에서 종점을 제거한다.")
+    @Test
+    void 노선에서_종점_제거() {
+        // given
+        JsonPath 역삼역 = 지하철_역_생성("역삼역");
         Map<String, Object> paramMap = new HashMap<>();
-        paramMap.put("upStationId", getId(신도림역));
-        paramMap.put("downStationId", getId(대림역));
+        paramMap.put("upStationId", getId(강남역));
+        paramMap.put("downStationId", getId(역삼역));
         paramMap.put("distance", 5);
```

Comment:
이 부분도 작성해주신 것 처럼 아래와 같이 해볼 수 있지 않을까 합니다 :D
```suggestion
        Map<String, Object> paramMap = 지하철_구간_추가_파라미터_생성(getId(강남역), getId(역삼역), 5);
```

Reply:
만들어놓고 써먹지를 않았군요 ㅎㅎ 변경하겠습니다!